### DISCUSSION TITLE
User Playlists

### DISCUSSION THREAD

**Discussion by ghost** [lutris/lutris#4156] [Ideas]
I think it would be greatly beneficial for Lutris to have the option of creating our own playlists. At the moment, the only “playlist” that can be used is the “Favorites” one.

For example, I’d like to make a playlist called “Christmas” for games that I like to play around Christmas time, or a playlist called “high school” for games that allow me to relive childhood memories. How about “completed” for games that I have beaten. Even a playlist called “not working” for games that I haven’t been able to run on Linux yet.

Not having the ability to make playlists is the last obstacle to overcome before being able to ditch Windows completely and go full-on Linux.

Come to think of it, the ability to create labels and be able to filter with them would also work well.

Thank you for your time.

**Comment by sallymanda** [comment]
I second this, would just like to be able to put things in cooperative and single player playlists as thats the primary thing I'm looking for when I want to play something with my boyf.

**Comment by jurassicjordan** [comment]
i also second this, I'm a seasonal games lover, and  want to sort my horror games out for October, a playlist would be perfect for this
